ProxyAgent/appsettings.ini

```diff
@@ -2,6 +2,7 @@
 ; The remote endpoint where requests are forwarded to
 ; The REMOTE_ENDPOINT env var is set on PCS deployments
 endpoint = "${REMOTE_ENDPOINT}"
+thumbprint = "${REMOTE_ENDPOINT_SSL_THUMBPRINT}"
```

Comment:
nit: rename to "ssl_cert_thumbprint"

Reply:
Won't rename this since it requires touching a bunch of files for deployment.